addons/mail/models/mail_activity.py

```diff
@@ -592,7 +592,6 @@ class MailActivityMixin(models.AbstractModel):
     Activities come with a new JS widget for the form view. It is integrated in the
     Chatter widget although it is a separate widget. It displays activities linked
     to the current record and allow to schedule, edit and mark done activities.
-    Use widget="mail_activity" on activity_ids field in form view to use it.
```

Comment:
Note: this comment needs to be replaced with whatever we decide in our new syntax.

Reply:
Nothing is needed in fact, just need to have the field (for now)